crates/types/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "rjacraft_types"
+description = "Types for rjacraft"
```

Comment:
```suggestion
description = "Rjacraft's wrapper for working with various generated protocol ID constants from Minecraft"
```

Reply:
Done; see [d8fa342:/crates/types/Cargo.toml](https://github.com/rjacraft/rjacraft/blob/d8fa342630650f9e78acc4219e82bd18e281f618/crates/types/Cargo.toml#L3).

Reply:
Still doesn't really capture the purpose of the crate. Protocol constructs go in the `protocol` crate, not in here. This is about protocol IDs and identifiers.